pyproject.toml

```diff
@@ -7,3 +7,6 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 110
+
+[pytest]
+timeout = 60
```

Comment:
If this is 60 seconds, I think many of our tests would legitimately take longer than that? If we're setting a timeout we probably want to make this very long as a failsafe to end tests which clearly aren't going to finish running.

Reply:
From timing this locally,
$ pytest --durations=10
the longest test on my system is ~10s but I still don't think it hurts to set a longer extreme timeout of (e.g. 5 mins)